src/controllers/requestReview.ts

```diff
@@ -72,23 +99,35 @@ export const getRequestReviewsByPostingUser = async (posterId: string) => {
 };
 
 export const createRequestReview = async (
-  target: string,
-  poster: string,
+  request: string,
   comment: string,
   rating: number
 ) => {
   try {
-    const review = new RequestReview({
-      target,
-      poster,
-      comment,
-      rating,
-    });
-    await review.save();
-    return {
-      status: true,
-      data: review,
-    };
+    const populatedRequest = await Request.findOne({
+      _id: request,
+    }).exec();
+    console.log(populatedRequest);
```

Comment:
Remove console.log and use logger if needed

Reply:
Fixed